### DISCUSSION TITLE
Mosquitto: avviare applicazione su pc remoto tramite MQTT

### DISCUSSION THREAD

**Discussion by labattazza** [indomus/forum#2413] [Networking]
Ho un server dove gira Home Assistant supervised col suo bravo bloker MQTT.

Vorrei provare a sfruttarlo per avviare applicazioni su client remoti, che accedo solo quando ho bisogno, senza dover interagire con i comandi.

Ad esempio, sul pc linux collegato al monitor in soggiorno, vorrei avviare Spotify tramite un comando lanciato da Home Assistant, e poi comandarlo da telefono o in altro modo.

Il mio problema è che non trovo esempi da cui partire per capire come configurare il client su pc remot, e come creare il topic da lanciare.

Se aveste dei buoni spunti da studiare, ve ne sarei molto grato!
Grazie!

**Comment by snakuzzo** [comment]
Puoi farlo, ma sul PC remoto dovresti avere innanzitutto un client MQTT e poi creare uno script (o qualcosa di simile) che faccia quello di cui tu hai bisogno. Non è che hai tutto gratis solo avendo il broker :)
